Makefile

```diff
@@ -103,7 +103,7 @@ build_docs_pdf:
 
 .PHONY: docs_serve
 docs_serve: build_docs
-	podman run -v $(CURDIR)/docs:/docs:Z --rm -p 8088:8088/tcp $(DOCS_BUILD_CONTAINER) docs_serve 
+	podman run -it -v $(CURDIR)/docs:/docs:Z --rm -p 8088:8088/tcp $(DOCS_BUILD_CONTAINER) docs_serve 
```

Comment:
@anjannath `it` is for interactive mode, why do need the docs_serve to run as interactive session?

Reply:
So that we can terminate the http server process with `ctrl+c`